apps/agora/app/src/index.html

```diff
@@ -6,6 +6,30 @@
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="icon" type="image/x-icon" href="favicon.ico" />
+
+    <!-- Facebook Meta Tags -->
+    <meta property="og:url" content="https://agora.adknowledgeportal.org" />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="Agora" />
+    <meta
+      property="og:description"
+      content="Explore the role of human genes in Alzheimer’s Disease using interactive visualizations of high-dimensional multi-omic evidence, analyses, and target nominations from National Institute on Aging (NIA) researchers and the broader AD research community."
+    />
+    <meta property="og:image" content="https://agora.adkknowledgeportal.org/images/og-social.png" />
```

Comment:
Looks like there might be a typo?

```suggestion
    <meta property="og:image" content="https://agora.adknowledgeportal.org/images/og-social.png" />
```

Reply:
Also, does anything else need to be updated so the OG image will be available at this path (e.g. update Caddy to serve static files)? @tschaffter - how does openchallenges handle serving the OG image?

Reply:
Processed with Thumbor and served in S3

Reply:
Created a GitHub [repo](https://github.com/Sage-Bionetworks/agora-images) to host static images for Agora.